capa/features/extractors/vmray/__init__.py

```diff
@@ -45,9 +61,11 @@ def __init__(self, zipfile_path: Path):
         self.exports: Dict[int, str] = {}
         self.imports: Dict[int, Tuple[str, str]] = {}
         self.sections: Dict[int, str] = {}
-        self.os_pid_by_monitor_id: Dict[int, int] = {}
-        self.tids_by_pid: Dict[int, List[int]] = defaultdict(list)
-        self.process_calls: Dict[int, Dict[int, List[FunctionCall]]] = defaultdict(lambda: defaultdict(list))
+        self.monitor_processes: Dict[int, VMRayMonitorProcess] = {}
+        self.monitor_threads: Dict[int, VMRayMonitorThread] = {}
+        self.monitor_threads_by_monitor_process: Dict[int, List[int]] = defaultdict(list)
+        self.monitor_process_calls: Dict[int, Dict[int, List[FunctionCall]]] = defaultdict(lambda: defaultdict(list))
```

Comment:
I think two respective comments here would be useful to know what maps to what (pid and tid)

Reply:
https://github.com/mandiant/capa/pull/2396/commits/d331f22639fc83977d8707408498431336f50872